.mailmap

```diff
@@ -202,4 +203,27 @@ Tanay Gahlot <tanaygahlot@gmail.com> Tanay <tanaygahlot@gmail.com>
 Erkka Heinila <erkkahe@gmail.com> Teekuningas <erkkahe@gmail.com>
 Burkhard Maess <burkhard.maess@arcor.de> Burkhard Maess <bmaess@users.noreply.github.com>
 Lorenz Esch <Lorenz.Esch@tu-ilmenau.de> Lorenz Esch <lorenzesch@hotmail.com>
-Joris Van den Bossche <jorisvandenbossche@gmail.com>
+Joris Van den Bossche <jorisvandenbossche@gmail.com> Joris Van den Bossche <jorisvandenbossche@gmail.com>
+Christina Zhao <zhaotc@uw.edu> ChristinaZhao <zhaotc@uw.edu>
+Rotem Falach <falachrotem@gmail.com> Falach <Rotemfa0@gmail.com>
+Nicolas Gensollen <nicolas.gensollen@gmail.com> Gensollen <nicolas.gensollen@gmail.com>
+Matteo Anelli <matteo.anelli@aalto.fi> Matteo Anelli <matteanelli.96.11@gmail.com>
+Ram Pari <ramsbam@gmail.com> Ram <ramsbam@gmail.com>
+Steven Bierer <neurolaunch@gmail.copm> Steven Bierer <40672003+NeuroLaunch@users.noreply.github.com>
+Valerii Chirkov <vagechirkov@gmail.com> Valerii <42982039+vagechirkov@users.noreply.github.com>
+Zhi Zhang <850734033@qq.com> ZHANG Zhi <850734033@qq.com>
+Ana Radanovic <anaradanovica@gmail.com> anaradanovic <79697247+anaradanovic@users.noreply.github.com>
+Catalina Magalvan <79813952+catalinamagalvan@users.noreply.github.com> catalinamagalvan <79813952+catalinamagalvan@users.noreply.github.com>
+Christian Mista <cmista@ingenieria.uner.edu.ar> cmista <79416030+cmista@users.noreply.github.com>
+Felix Klotzsche <klotzsche@cbs.mpg.de> eioe <felix_klotzsche@web.de>
+Felix Klotzsche <klotzsche@cbs.mpg.de> eioe <klotzsche@cbs.mpg.de>
+Enrico Vara <69973551+enricovara@users.noreply.github.com> enricovara <69973551+enricovara@users.noreply.github.com>
+Giorgio Marinato <giorgio.marinato@unitn.it> neurogima <76406896+neurogima@users.noreply.github.com>
+Erica Peterson <nordme@uw.edu> nordme <nordme@uw.edu>
+Erica Peterson <nordme@uw.edu> nordme <38704848+nordme@users.noreply.github.com>
+Timon Merk <38216460+timonmerk@users.noreply.github.com> timonmerk <38216460+timonmerk@users.noreply.github.com>
+apadee <44297909+apadee@users.noreply.github.com> apadee <44297909+apadee@users.noreply.github.com>
+kimcoco <41998428+kimcoco@users.noreply.github.com> kimcoco <41998428+kimcoco@users.noreply.github.com>
+monkeyman192 <monkey_man_192@yahoo.com.au> monkeyman192 <monkey_man_192@yahoo.com.au>
+sfc-neuro <78911192+sfc-neuro@users.noreply.github.com> sfc-neuro <78911192+sfc-neuro@users.noreply.github.com>
+DataFox <sfox@riseup.net> DataFox <sfox@riseup.net>
```

Comment:
@larsoner or @drammock if you can help me get the true names of these people it would help

Reply:
I'll push what I find here

Reply:
done :)